### PR TITLE
Remove Placemat v1 compatible codes

### PR DESCRIPTION
**Pull request by ysksuzuki**
This PR removes placemat v1 compatible codes. Also, reverts Teleport test to run on the test host. 
The original teleport test modifies `/etc/hosts` using sed and it leads to the error when the test runs inside a k8s container. So this PR fixes it to stop using sed as well.

### CODE REVIEW

File: test/teleport_test.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"io/ioutil"
```
Comment (tapih) [inline test/teleport_test.go:11]: `io/ioutil` is deprecated in Go [1.16](https://golang.org/doc/go1.16#ioutil). Please use `os` instead.

File: test/teleport_test.go [hunk 2]
```diff
@@ -185,29 +180,21 @@ func teleportSSHConnectionTest() {
...
-
-	By("clearing teleport_cookie.txt")
-	_, stderr, err = ExecAt(boot1, "rm", filename)
-	Expect(err).ShouldNot(HaveOccurred(), "stderr=%s", stderr)
-
 	By("clearing /etc/hosts")
-	_, stderr, err = ExecAt(boot1, "sudo", "sed", "-i", "-e", "/teleport.gcp0.dev-ne.co/d", "/etc/hosts")
-	Expect(err).ShouldNot(HaveOccurred(), "stderr=%s", stderr)
+	// /etc/hosts cannot be modified via sed directly inside a container because that is a mount point
+	b, err = ioutil.ReadFile("./hosts")
```
Comment (tapih) [inline test/teleport_test.go:194]: This can be purged if we use `curl --resolve`.

**Review by tapih — CHANGES_REQUESTED** [review]

File: test/teleport_test.go (hunk 2 above)
Comment (ysksuzuki) [reply test/teleport_test.go]: I though I could, but unfortunatelly `tsh` seems to require `/etc/hosts` entry.

**Review by tapih — APPROVED** [review]
LGTM

**Pull request merged**